wandb/sdk/wandb_run.py

```diff
@@ -971,8 +973,8 @@ def _config_callback(
     def _config_artifact_callback(
         self, key: str, val: Union[str, Artifact]
     ) -> Union[Artifact, public.Artifact]:
-        if isinstance(val, string_types) and val.startswith("wandb-artifact://"):
-            artifact_string = val[len("wandb-artifact://") :]
+        if isinstance(val, str) and val.startswith("wandb-artifact"):
```

Comment:
Should be really rare, but I imagine a user could set a config value to "wandb-artifact-foo" and the parser would blow up.

Reply:
I see now, would be good to also check `and "://" in val`

Reply:
Fixed